app/lib/backend_api_logic/service_extension.rb

```diff
@@ -17,28 +19,37 @@ module ServiceExtension
       end, class_name: 'Metric'
     end
 
-    def first_backend_api
-      @backend_api ||= find_or_create_first_backend_api!
+    class BackendApiProxy
+      attr_reader :service
+      delegate :account, :backend_apis, :backend_api_configs, to: :service
+
+      def initialize(service)
+        @service = service
+      end
+
+      def backend_api_config
+        @backend_api_config ||= backend_api.backend_api_configs.first ||
+                                backend_api_configs.build(path: '', backend_api: backend_api)
+      end
+
+      def backend_api
+        return unless account
```

Comment:
Please avoid returning `nil` object. This is just breaking NullObject pattern
why is account nil?